docs/src/Text.doc.js

```diff
@@ -47,6 +47,13 @@ card(
         defaultValue: false,
         href: 'styles',
       },
+      {
+        name: 'lineClamp',
+        type: 'number',
+        description:
+          'Visually truncate the text to the specified number of lines. This also adds the `title` attribute if `children` is a string.',
```

Comment:
This also adds the `title` attribute if `children` is a string.', This could be a little more educational, it might not be obvious what it means for some.